docs/learning/30-jpa-pgvector-type-mapping.md

```
@@ -0,0 +1,167 @@
+void saveWithEmbedding(@Param("embedding") String embedding);
+```
+
+결과: `SchemaManagementException` 발생. Hibernate의 `validate` 모드가 DB 스키마와 Entity를 대조할 때, `@Transient` 필드는 무시하지만 **DB에 있는 `embedding` 컬럼에 매핑되는 Entity 필드가 없다**고 에러를 던진다. 반대 방향의 문제.
+
+`@Transient`를 빼면 이번에는 Hibernate가 `vector` 타입을 몰라서 또 에러. 진퇴양난.
+
+### B. stringtype=unspecified — 동작하지만 위험
+
+JDBC URL에 `?stringtype=unspecified`를 붙이면, PostgreSQL JDBC 드라이버가 String 파라미터를 보낼 때 타입을 지정하지 않는다. PostgreSQL 서버가 알아서 타입을 추론해준다.
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)